eureka-client-archaius2/build.gradle

```diff
@@ -1,7 +1,14 @@
 apply plugin: 'nebula.test-jar'
 
+def archaius2Version = '2.1.4'
+
 dependencies {
-    compile project(':eureka-client')
+    compile (project(':eureka-client')) {
+        // archaius1
+        exclude group: 'com.netflix.archaius', module: 'archaius-core'
```

Comment:
If eureka-client depends directly on archaius-core, then this exclude will not prevent them from getting it. Excludes are not transitive, so while archaius1 may not appear in this project, it will be present for everyone who depends on it. We may need to consider refactoring eureka-client to shift this dependency to another place.